doc/user/layouts/partials/head.html

```diff
@@ -115,35 +115,50 @@
 
 {{/* Tabs */}}
 <script>
-  $(document).ready(function () {
-    // make nav-tab lists from tab-panes
-    $(".tab-content").each(function (idx, tab) {
-      $(tab)
-        .find(".tab-pane")
-        .each(function (item) {
-          var navTabs = $(this).closest(".code-tabs").find(".nav-tabs"),
-            title = $(this).attr("title"),
-            id = title
-              .toLowerCase()
-              .replace(/ /g, "-")
-              .replace(/[^\w-]+/g, "");
-          navTabs.append(`<li><a href="#${id}-t${idx}">${title}</a></li>`);
-        });
-    });
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
some housecleaning for tabs.
Changed from jquery to regular js and allow for nested tabs.
Decided against using nested tabs but figured it won't hurt to have it for future use.